sdk/servicebus/service-bus/src/clientEntityContext.ts

```diff
@@ -79,6 +79,16 @@ export interface ClientEntityContextBase {
    * @property {ClientType} [clientType] Type of the client, used mostly for logging
    */
   clientType: ClientType;
+
+  /**
+   * @property {string} [clientId] Unique Id of the client for which this context is created
+   */
+  clientId: string;
+
+  /**
+   * @property {boolean} [isClosed] Denotes if close() was called on this client.
```

Comment:
For the new properties we should not add the `{boolean} [isClosed]` text as that is being removed in a different PR

Reply:
This is an internal class so will not be picked up for documentation. Didnt want to touch this as this class has too many other properties following the same format. Tried to keep changes minimal in the PR